ouroboros-consensus/src/unstable-consensus-testlib/Test/Util/Orphans/Arbitrary.hs

```diff
@@ -417,3 +422,11 @@ instance Arbitrary a => Arbitrary (LoE a) where
   arbitrary = oneof [pure LoEDisabled, LoEEnabled <$> arbitrary]
   shrink LoEDisabled    = []
   shrink (LoEEnabled x) = LoEDisabled : map LoEEnabled (shrink x)
+
+{-------------------------------------------------------------------------------
+  SecurityParam
+-------------------------------------------------------------------------------}
+
+instance Arbitrary SecurityParam where
+  arbitrary = SecurityParam <$> choose (0, 6)
+  shrink (SecurityParam k) = SecurityParam <$> shrink k
```

Comment:
A security parameter of `0` is rather fishy, e.g. lots of derived quantities like `3k/f` would then also be `0`. I think it makes sense to increase the lower bound to `1`, or even remove this instance and locally define a generator in the LedgerDB tests where it is currently used.